src/Pages/Home.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react';
+import Hello from "../Components/Hello";
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
:art: Can we conver this to hooks? We haven't actually had a real discussion about this, but I think hooks are preferred over state constructors.